lib/babe/babe_integration_test.go

```diff
@@ -408,4 +408,3 @@ func TestService_PauseAndResume(t *testing.T) {
 	err = bs.Stop()
 	require.NoError(t, err)
 }
-
```

Comment:
What's wrong with trailing new lines? Let's think about CLI people using `cat` or other poor man's editor :smile:

Reply:
ah you are so right! I think goland format did it to me